include/framework/runtime.hpp

```diff
@@ -81,6 +81,7 @@ inline constexpr auto MQTT_EXTERNAL_PREFIX = "";
 inline constexpr auto TELEMETRY_PREFIX = "everest-telemetry";
 inline constexpr auto TELEMETRY_ENABLED = false;
 inline constexpr auto VALIDATE_SCHEMA = false;
+inline constexpr auto VALIDATE_SCHEMA_ON_STARTUP = false;
```

Comment:
For the user it might be hard to guess what the 'validate_schema' and 'validate_schema_on_startup' is doing.  So maybe we can find a better naming here.

Reply:
Do you have a suggestion? I did not have a better idea